entrypoint.sh

```diff
@@ -27,13 +37,14 @@ EOF
   
   echo "Response Body: $(cat $filePath)"
   echo "Response Status Code: $STATUS_CODE"
-  DEPLOYMENT_ID=$(cat $filePath | jq -r '.deployment_id')
   
-  if [[ -z "$DEPLOYMENT_ID"  || $STATUS_CODE != 2* ]];
+  
+  if [[ $STATUS_CODE != 2* ]];
   then
     echo "Failed to deploy app, err: $(cat $filePath)"
     exit 1
   fi
+  DEPLOYMENT_ID=$(cat $filePath | jq -r '.deployment_id')
```

Comment:
why is this moved?

Reply:
deployment id is not available if status wasn't 2XX, hence moved it after 2XX check